WikiWatershed/model-my-watershed
# Make App Work With tr55 Updates From Stroud

Stroud made a lot of [updates to the tr55 model](https://github.com/WikiWatershed/tr-55/pull/66), so we've released a new version (1.2.0). I've updated model-my-watershed to use the new version, and fixed the inconsistency that followed - one of the Stroud updates was to change the bmps landcover key `infiltration_trench` to `infiltration_basin`.

## Testing

- Pull this branch

---
(If you already have mmw up and running)
- `vagrant provision worker` (I've had trouble with pip not updating tr55, so `vagrant ssh worker` and confirm that the tr55 version is now 1.2.0 with `pip show tr55`.  
- `./scripts/bundle.sh`

----

- In your browser go to `localhost:8000`
- Draw some area
- In the Analyse Menu, click Model --> Site Storm Model
- Click the "+" to add a new scenario and then "+ Conservation Practice" --> Veg Basin
- Draw a Veg Basin
- Confirm something shows up for the model and that there are no errors.

Connects [67](https://github.com/WikiWatershed/tr-55/issues/67)
